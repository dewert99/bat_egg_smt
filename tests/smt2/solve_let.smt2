(declare-const x Bool)
(assert x)
(check-sat-assuming ((let ((x false)) x) x))
(get-unsat-core)
(check-sat-assuming (x))
(check-sat-assuming ((let ((x false) (x x)) x)))