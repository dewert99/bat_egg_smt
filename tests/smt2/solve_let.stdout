unsat
((let ((x false)) x))
sat
sat
